stats.go

```diff
@@ -0,0 +1,92 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package datadog
+
+import (
+	"fmt"
+	"log"
+	"sync"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"go.opencensus.io/stats/view"
+)
+
+// collector implements statsd.Client
+type statsExporter struct {
+	opts     Options
+	mu       sync.Mutex // mu guards viewData
```

Comment:
Please move this above `viewData`, it should pair with the field that it guards.